docs/config/README.md

```diff
@@ -1755,6 +1756,39 @@ By default the module will be shown if any of the following conditions are met:
 symbol = "∴ "
 ```
 
+## localip
+
+The `localip` module shows the IPv4 address of the primary network interface.
+
+### Options
+
+| Option     | Default                     | Description                                                                                                                          |
+| ---------- | --------------------------- | ------------------------------------------------------------------------------------------------------------------------------------ |
+| `ssh_only` | `true`                      | Only show IP address when connected to an SSH session.                                                                               |
+| `format`   | `"[$localipv4]($style) "`     | The format for the module.                                                                                                           |
+| `style`    | `"bold yellow"`              | The style for the module.                                                                                                            |
+| `disabled` | `false`                     | Disables the `localip` module.                                                                                                      |
+
+### Variables
+
+| Variable    | Example      | Description                          |
+| --------    | -------      | ------------------------------------ |
+| localipv4\* | 192.168.1.13 | Contains the primary IPv4 address    |
```

Comment:
```suggestion
| localipv4 | 192.168.1.13 | Contains the primary IPv4 address    |
```

Reply:
Here we go, tnx for the patience and your time 🙏